Operations.podspec

```diff
@@ -15,22 +15,24 @@ session Advanced NSOperations: https://developer.apple.com/videos/wwdc/2015/?id=
   s.module_name       = 'Operations'
   s.social_media_url  = 'https://twitter.com/danthorpe'
   s.requires_arc      = true
-  s.platform          = :ios, '8.0'
   s.default_subspec   = 'Base'
+  s.ios.deployment_target = "8.0"
+  s.osx.deployment_target = "10.10"
 
   s.subspec 'Base' do |ss|
     ss.source_files      = 'Operations/**/*.{swift,m,h}'
```

Comment:
@danthorpe BTW, this rule also greedily includes AddressBook stuff. Should I also fix that?